listing-rewards/program/src/state/metaplex_anchor.rs

```diff
@@ -0,0 +1,78 @@
+use anchor_lang::prelude::*;
```

Comment:
Can you link to the file in bubblegum program with a comment saying that this was extracted from that file and should be referenced for further updates to TokenMetadata account deserialization.

Reply:
Sure thing :)